android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerViewManager.java

```diff
@@ -94,14 +104,42 @@ public void onDropViewInstance(ReactExoplayerView view) {
         );
     }
 
+    @ReactProp(name = PROP_DRM)
+    public void setDRM(final ReactExoplayerView videoView, @Nullable ReadableMap drm) {
+        Log.d("SetDRM", "Set DRM config");
```

Comment:
Debug logging?

Reply:
@danielmarino24i is there a reason for this logging in Android?

Reply:
Yep, I forgot to remove the logs, I will delete them

Reply:
updated